lyluatex.lua

```diff
@@ -910,4 +946,34 @@ function ly.set_property(k, v)
 end
 
 
+function ly.verbprint(lines, highlight)
+    local spaces
+    if highlight then
+        --[[ We unfortunately need an external file,
+             as minted is a verbatim environment. ]]
+        local fname = ly.get_option('tmpdir')..'/verb.tex'
+        local f = io.open(fname, 'w')
+        f:write(
+            -- We use tex until one decedes to implement lilypond
+            '\\begin{minted}{tex}\n'..
```

Comment:
I'm not sure that forcing `minted` for highlighting is the correct approach.  `pygments` (the highlighter that `minted` is based on) doesn't currently support LilyPond code (as you note in your comment) and there are other syntax highlighting packages (`listings`, for one).  Further, it could be that someone will make use of the functioning syntax highlighter in `python-ly` to write a custom package for highlighting LilyPond code, rather than writing adapting an existing LaTeX package to support a new language.  I was simply using `minted` as an example of a package which can do syntax highlighting.  I think it would be better if we had a internal dummy environment which we alias to `noindent` (or perhaps `verbatim`) by default, but which a user could reassign to a LilyPond syntax highlighting environment defined elsewhere (by whatever method).

This would then nullify the comment below about how and wen to load `minted`.